commodore/helpers.py

```diff
@@ -163,6 +164,14 @@ def kapitan_compile(
     )
 
 
+def kapitan_inventory(config: Config) -> Dict:
+    """
+    Reset reclass cache and render inventory
+    """
+    reset_reclass_cache()
+    return inventory_reclass(config.inventory.inventory_dir)
```

Comment:
Shouldn't this return the "nodes" value?
```suggestion
    return inventory_reclass(config.inventory.inventory_dir)["nodes"]
```

Reply:
Not with the current implementation of `dependency_mgmt._discover_components()` but I've now refactored that to use the `applications` array of target `cluster` instead of the top-level `applications` dictionary keys